scripts/ilks2ascii.sh

```diff
@@ -1,11 +1,19 @@
-#!/bin/bash
-### ilks2ascii -- list ilks from ilk-registry in ascii
-### Usage: ./ilks2ascii <ilk-registry-address>
+#!/usr/bin/env bash
+### ilks2ascii -- list ilks from the IlkRegistry in ascii
+### Usage: ./ilks2ascii.sh
 
-ILKS=$(seth call "$1" "list()(bytes32[])")
+set -e
 
-for ilk in $(echo "$ILKS" | sed "s/,/ /g")
-do
-	seth --to-ascii "$ilk"
+[[ "$ETH_RPC_URL" ]] || { echo "Please set a ETH_RPC_URL"; exit 1; }
+
+CHANGELOG=0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F
+
+ILK_REGISTRY=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "ILK_REGISTRY")")")
```

Comment:
Getting that gas price error with seth here. If forge doesn't resolve the issue we should maybe see if we can file an issue with them to not rely on your local wallet balance for `call`s.

```
bmcmichael@bmcx1e:~/makerdao/ilk-registry$ ILK_REGISTRY=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "ILK_REGISTRY")")")
seth-rpc: {"id":1,"jsonrpc":"2.0","method":"eth_call","params":[{"data":"0x21f8a721494c4b5f52454749535452590000000000000000000000000000000000000000","from":"0xC0dE1AB51082442f821bA273273B4F7d32521D8b","gas":"0x1312d00","maxFeePerGas":"0x5d21dba000","maxPriorityFeePerGas":"0x77359400","to":"0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F"},"latest"]}
seth-rpc: error:   code       -32000
seth-rpc: error:   message    err: insufficient funds for gas * price + value: address 0xC0dE1AB51082442f821bA273273B4F7d32521D8b have 1691614669368968099 want 8000000000000000000 (supplied gas 20000000)
```

Reply:
I've addressed the seth gas price error, following previous [pattern](https://github.com/makerdao/spells-goerli/blob/master/scripts/wards.sh#L8) used in the `wards.sh` script in 5338bfb.